webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/UsersMessagesADM.scala

```diff
@@ -933,7 +938,7 @@ case class UserUpdatePayloadADM(username: Option[String] = None,
 object UsersADMJsonProtocol extends SprayJsonSupport with DefaultJsonProtocol with ProjectsADMJsonProtocol with GroupsADMJsonProtocol with PermissionsADMJsonProtocol {
 
     implicit val userADMFormat: JsonFormat[UserADM] = jsonFormat13(UserADM)
-    implicit val createUserApiRequestADMFormat: RootJsonFormat[CreateUserApiRequestADM] = jsonFormat8(CreateUserApiRequestADM)
+    implicit val createUserApiRequestADMFormat: RootJsonFormat[CreateUserApiRequestADM] = jsonFormat(CreateUserApiRequestADM, "id", "username", "email", "givenName", "familyName", "password", "status", "lang", "systemAdmin")
```

Comment:
Did you change this because `jsonFormat8` didn't work after you added `id`? If so, I think you can just use `jsonFormat9` instead.

Reply:
When I added `id, I changed it to `jsonFormat9`, but I got a function overloaded error asking for fieldnames. giving fieldnames in this way and using `jsonFormat` worked.

Reply:
OK!